data_rentgen/db/repositories/operation.py

```diff
@@ -25,7 +25,7 @@ async def create_or_update_bulk(self, operations: list[OperationDTO]) -> list[Op
             set_={
                 "name": func.coalesce(insert_statement.excluded.name, Operation.name),
                 "type": func.coalesce(insert_statement.excluded.type, Operation.type),
-                "status": func.coalesce(insert_statement.excluded.status, Operation.status),
+                "status": func.greatest(insert_statement.excluded.status, Operation.status),
```

Comment:
There is no chance that operation will have 'UNKNOWN' status after it's started?

Reply:
`UNKNOWN` status is set in case when we receive event containing run(id=1) + parentRun(id=1), but we don't receive dedicated run(id=1) with status of parent. So we create run(id=1, created_at=..., status=UNKNOWN)`, and that's all. If we already received paren run event, or receive it later, we replace `UNKNOWN` status with a received one.